lib/mcb.rb

```diff
@@ -195,6 +195,38 @@ def apiv1_opts(opts)
       opts
     end
 
+    # Return options necessary to connect to API V2.
+    #
+    # The opts passed in are examined determine which opts need to be added,
+    # this function essentially just fills in any missing options.
+    #
+    #   opts = apiv2_opts(opts)
+    def apiv2_opts(opts)
+      opts[:url] = opts[:url]
+      opts[:'max-pages'] = opts[:'max-pages']
+      opts[:token] = opts[:token]
+      opts[:all] = opts[:all]
```

Comment:
The way opts is being used here is confusing, is the intent to ensure these things have been specified or something else?